### ISSUE TITLE
What is your license status on this mod?

### ISSUE THREAD

**Issue by ScottoMotto** [Wyn-Price/betterunderground#12]
I saw your post on the Wild Caves 3 forum topic requesting permission for the port. Additional permission is not necessary. The author of WC3 has already indicated that the code is open source by word and deed:

https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/wip-mods/1442430-1-6-x-forge-wildcaves-3-v0-4-3-3?comment=346

https://github.com/AlexmaniaWardz/WildCaves3

Also, I would venture to say that your port is based on this one from GoToLink:

https://github.com/GotoLink/WildCaves3

That port has an open license stated:

https://github.com/GotoLink/WildCaves3/blob/master/LICENSE.md

Basically, I am asking for permission to continue forward with your port. The license you ported under requires that you pass on the license you received, which is an open one. But I respect your efforts.

Since you seem not to be interested in this mod and the current manager has pulled the mod repeatedly, I would like to continue this mod under the name Wild Caves 4 which is the proper one given its history. I plan to add new content and textures as well. Do you agree?

**Comment by ghost** [comment]
hey ah, i was going to upload the mod again, was a bit busy. today ill upload it with the GPLv2 licence. if you want to help the development, we can accept you well. my nick is extrememoddingmachine on curseforge. just contact me there

**Comment by ScottoMotto** [comment]
So where is the upload after one week? You are not proving to be a very reliable person.

**Comment by ghost** [comment]
oops, i forgot to write anything to here, sorry.
i'm banned from curse fucking again and i won't mod anymore. just do it as you wish, but ask also @Wyn-Price. but be careful about porting mods to new versions of minecraft, they ban you instantly if all the things arent ok on your side. it's what i experienced :D now i cant even upload my own mod(harshen universe) on a new account. they ban it instantly :D ...

**Comment by ScottoMotto** [comment]
@Wyn-Price so kenijey is off this and I'm sorry to hear that. It is definitely unfair in the case of this mod as the license is clear. I would like to proceed with this and try to get pre-approval from the moderators at Curse with the documentation on the license to back it up.

Again, I could port this myself probably. But there would always be doubt of the port's origins and why duplicate work? I do want to add new content.

**Comment by Wyn-Price** [comment]
Yeah sure you can port this. I haven't touched the code for a while, your free to do anything with it. Monetize it, add stuff, remove stuff go for it. It's all yours
